.idea/watcherTasks.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Yes, it is. Send me the ref, I didn't want to mess with gitignore until i knew what i was doing. I thought I left all the ide files untracked but this one must have been added accidentally.